java-api-bindings/install_dependencies_and_build.sh

```diff
@@ -0,0 +1,178 @@
+#!/bin/bash
+# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+USAGE="
+usage: install_dependencies_and_build.sh [options]
+
+Installs Maven, Java JDK and builds Tritonserver Java bindings
+-h|--help                  Shows usage
+-t|--triton-home           Expected Trition library location, default is: /opt/tritonserver
+-b|--build-home            Expected build location, default is: /tmp/build
+-v|--maven-version         Maven version, default is: "3.8.4"
+-c|--core-tag              Tag for core repo, defaut is: "main"
+-j|--jar-install-path      Path to install the bindings .jar
+--javacpp-branch           Javacpp-presets git path, default https://github.com/bytedeco/javacpp-presets.git
+--javacpp-tag              Javacpp-presets branch tag, default "master"
```

Comment:
"default is: " for javacpp-branch and javacpp-tag for consistency with above. Or any format, would just be good to keep it consistent. Also a typo on "default" for core-tag.